docs/reference.rst

```diff
@@ -949,6 +949,30 @@ Stores the number of milliseconds to wait between request retries.  By default,
 
 In the case that both this environment variable and the config's ``rpc_attempt_interval`` property are set, the value in ``rpc_attempt_interal`` will be used.
 
+
+SHOTGUN_API_DISABLE_ENTITY_OPTIMIZATION
+=======================================
+
+When set to ``1``, this environment variable will disable the entity optimization feature.  This feature is enabled by default and is used to reduce the payload size made to the server when retrieving entities improving performance.  
+
+For example, a ``find`` call like this:
+
+.. code-block:: python
+
+    sg.find('Asset', [['project', 'is', {'created_at': datetime.datetime(2015, 12, 16, 11, 2, 10, tzinfo), 'id': 9999, 'name': 'Demo: Game', 'type': 'Project', ...}]])
```

Comment:
Could you add some line split in this code please so it's readable without horizontal scrolling